project-portfolio

Code-lab-web opened the pull request:
Please include a link to your Figma design and a Netlify link.

Code-lab-web committed:
Main
Please include a link to your Figma design and a Netlify link.

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Implement styled-components for global styles, theme, and responsive breakpoints; refactor App component and add tests

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
git commit -m "Add card component and header component"

Code-lab-web committed:
Fix: Resolve issue with App.jsx rendering logic

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Refactor App component, update README, and enhance styled-components integration; add password validation in new Flask API

Code-lab-web committed:
Enhance App component: import React and styled-components, update index.html title, and add theme usage in ReactHook

Code-lab-web committed:
Git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
Code-lab-web

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
Merge remote-tracking branch 'refs/remotes/origin/main'

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-web-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

JennieDalgren reviewed (changes requested):
Read the reqiurements for this project and follow the guide on fow to work with github! It is not possible to review this at all.

You are not approved on this project

Commit:
git commit -m "Code-lab-web"

HIPPIEKICK commented:
ping

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
Create SECURITY.md for security policy

Added a security policy document outlining supported versions and vulnerability reporting.

Code-lab-web committed:
Add CodeQL analysis workflow configuration

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add CRDA static analysis workflow

This workflow performs static analysis of source code using Red Hat CodeReady Dependency Analytics, triggered on pushes, pull requests, schedules, and manually.

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"
